src/model.jl

```diff
@@ -232,11 +260,11 @@ function _add_constraint_power_balance!(
     end
     if demand_flexibility.enabled
         injections = JuMP.@expression(
-            m, injections + sets.load_bus_map * m[:load_shift_dn]
+            m, injections + sets.flexible_load_bus_map * m[:load_shift_dn]
         )
         withdrawals = JuMP.@expression(
-            m, withdrawals + sets.load_bus_map * m[:load_shift_up]
-        )
+            m, withdrawals + sets.flexible_load_bus_map * m[:load_shift_up]
+        )                
```

Comment:
Unnecessary whitespace change.

Reply:
@danielolsen I should have fixed the white space problem in the last commit. I am using jupyter to run julia codes and the editor does not visually distinguish white spaces from the background so I didn't notice, sorry for that. Also I apologize for addressing these comments so late, I spent all the time in the past 10 days on working on a TSG special issue submission of which the deadline just passed..

Reply:
Good luck on the submission! PR #156 will introduce automated style checking, so this should be more obvious in the future.